### ISSUE TITLE
The proper way to show large images in RecyclerView

### ISSUE THREAD

**Issue by ealymbaev** [facebook/fresco#765]
My use case:

- I have a RecyclerView with image items. The images are quite large. (let's say 1mb each)
- If image is downloaded (cached) - they are shown directly. Otherwise only download button is shown.
- When user clicks download button - I just set the image uri to the SimpleDraweeView and Fresco downloads and shows the image
- Next time I just check the Fresco cache and show image (if is in cache) or download button
- I also show full size image when user clicks downloaded image.

The Problem:

- Downlaoded (cached by Fresco) images are loaded slowly when scrolling the list.
- I understand why this is happening, but I need another way to show images quickly.

Is there any way to create smaller thumbnails on the fly and cache them as well? BUT please note that I need to be sure that both thumbnail and the original full size image are in the cache at the same time.

In other words - when using the app offline - if the images is in the list, user should be able to view full size image as well.

**Comment by IanChilds** [comment]
You can create smaller thumbnails (and cache them) by specifying resize params for your image. This will have a different cache key to the same url with no resize params.
